storage/s3/src/server/wrappers/S3ClientWrapper.ts

```diff
@@ -99,6 +99,46 @@ export class S3ClientWrapper {
     await upload.done();
   }
 
+  public async listDirectories(options?: {
+    maxResults?: number;
+    includeEmptyFiles?: boolean;
```

Comment:
I left this because I didn't know if we wanted to expose this option parameter or not. In that case I would add it in the base interface. Should I add options or not (I personnaly don't need it).

Reply:
It would be better to not have any code that we do not use. The only interfaces that should be directly consumed by users is `ServerStorage`, `ClientStorage` and `FrontendStorage`. When designing those abstract classes we do think about what users might/might not need. Any other helper classes (like `S3ClientWrapper` for example) are exported from packages only so that the users could construct them when constructing storage classes manually. Their interface is kept as minimal as possible and we only add additional things to satisfy the requirements of storage classes. So to answer the question I think we should remove them.